archivebox/cli/archivebox_add.py

```diff
@@ -62,10 +62,16 @@ def main(args: Optional[List[str]]=None, stdin: Optional[IO]=None, pwd: Optional
         help="Re-archive URLs from scratch, overwriting any existing files"
     )
     parser.add_argument(
-        '--init', #'-i',
+        "--init", #'-i',
         action='store_true',
         help="Init/upgrade the curent data directory before adding",
     )
+    parser.add_argument(
+        "--extract",
+        nargs="+",
```

Comment:
Instead of using nargs, can you make it take a single arg as a csv list so that the argument parsing is consistently 1 flag + 1 arg:

`--extract=wget,media,git` or `--extract wget,media,git`

Reply:
@pirate done!